test/data.c

```diff
@@ -196,6 +196,9 @@ char message[25][80] = /**< Array of CW/DigiMode messages
     "", "", "", "", "", "", "", "", "", "", ""
 };
 
+char fkey_header[60] =
+    "   1=CQ  2=DE  3=RST 4=73  5=HIS  6=MY  7=B4   8=AGN  9=?  ";
+
```

Comment:
We could drop the leading and trailing spaces as they are inserted anyway.

Reply:
Done. Just had to reverse left and right padding to keep the current display.
(odd number of spaces has to divided into two)

Reply:
Nice job, thank you very much. Small problem, when you refresh using ":cfg" command, hdrline will be pushed to the left without a single leading blank ;-) .

Reply:
:CFG has a general issue, not only for this config: re-reading of config file is not working correctly. To be fixed later.

Reply:
Easy to fix here. Just call `center_fkey_header` on line 331 of changepars.c (just after rereading config and rules file).

Reply:
Re-reading is conceptually borken. E.g. will not release and reopen resources like sockets (say what if one changes the rig model number?). I wouldn't consider touching this now.

Reply:
Sorry, I do not agree here. Even if its broken does not mean we should make the situation worse. 

We should either make it right here or if :CFG is sooo bad - than we should just drop or block it completely.